backend/fms_core/template_importer/importers/_generic.py

```diff
@@ -1,12 +1,19 @@
 from django.core.exceptions import ValidationError
+from django.conf import settings
 from pandas import pandas as pd
 from django.db import transaction
+import time
 import reversion
+import os
 
 from ..sheet_data import SheetData
 from .._utils import blank_and_nan_to_none
 from fms_core.utils import str_normalize
 
+from fms_core.models import ImportedFile
+
+UPLOAD_PATH = os.path.join(settings.MEDIA_ROOT, 'uploads/')
```

Comment:
Maybe this should be defined in settings.py? Also, I'd be tempted to name this UPLOADED_TEMPLATES_PATH, and name the folder under /media "templates", just in case freezeman has other types of uploads in the future.

Reply:
Excellent suggestions ... yep